### PR TITLE
adding my webblocks to the project

### PR DESCRIPTION
**Pull request by dhiraj1008**
please add to the list @royranger

### CODE REVIEW

**Comment by sophiabrandt** [comment]
Please remove git submodule.

Please provide license to the images you used to make sure that we don't violate copyright by merging your PR.

File: src/blocks/after/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<html>
+    <head>
+    	 <meta name="viewport" content="width=device-width, initial-scale=1">
+          <title>Animation of dog,cat and rat</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+	</head>
+    <body background="bg.jpg">
```
Comment (sophiabrandt) [inline src/blocks/after/index.html:7]: You need to commit the images to git, too, or this won't work. Or link to a public URL.

File: src/blocks/after/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+body{
+          background-repeat: no-repeat;
+          background-size: 100% 100%;
+          background-attachment: fixed;
+}
+
```
Comment (sophiabrandt) [inline src/blocks/after/style.css:6]: Nitpick: please remove all the empty lines.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by MattCSmith** [comment]
Closing due to unresolved change request. Feel free to resolve and reopen.

**Pull request closed without merging**